content/desktop/install/windows-install.md

```diff
@@ -27,14 +27,15 @@ aliases:
 
 This page contains the download URL, information about system requirements, and instructions on how to install Docker Desktop for Windows.
 
-{{< button text="Docker Desktop for Windows" url="https://desktop.docker.com/win/main/amd64/Docker%20Desktop%20Installer.exe" >}}
+{{< button text="Docker Desktop for Window with Intel chip" url="https://desktop.docker.com/win/main/amd64/Docker%20Desktop%20Installer.exe?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-win-amd64" >}}
+{{< button text="Docker Desktop for Windows with Apple silicon (Beta)" url="https://desktop.docker.com/win/main/arm64/Docker%20Desktop%20Installer.exe?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-win-arm64" >}}
 
 _For checksums, see [Release notes](../release-notes.md)_
 
 ## System requirements
 
 {{< tabs >}}
-{{< tab name="WSL 2 backend" >}}
+{{< tab name="WSL 2 backend (Intel)" >}}
```

Comment:
```suggestion
{{< tab name="WSL 2 backend (x86_64)" >}}
```

Reply:
It works on Mac because with those devices you have either an Intel CPU or Apple Silicon. But on Windows, x86 is more appropriate since the vendor can be either Intel or AMD. It's the architecture we refer to here.